lib/event_processor/v1/v1EventProcessor.ts

```
@@ -83,7 +83,9 @@ export class LogTierV1EventProcessor implements EventProcessor {
       const dispatcher = useClosingDispatcher && this.closingDispatcher
         ? this.closingDispatcher : this.dispatcher;
 
-      dispatcher.dispatchEvent(formattedEvent, () => {
+      // TODO: this does not do anything if the dispatcher fails 
+      // to dispatch. What should be done in that case?
```

Catch and warn not sufficient?

---

currently, all sdks ignore this error I think. But imo, there should at least be a retries.